src/main/java/leo/client/Browser.java

```diff
@@ -14,33 +14,34 @@
 import javax.swing.*;
 import java.lang.reflect.Method;
 
-public class Browser {
-	private static final String errMsg = "Error attempting to launch web browser";
 
-	public static void open(String url) {
-		String osName = System.getProperty("os.name");
+public class Browser {
+    private static final String errMsg = "Error attempting to launch web browser";
 
-		try {
-			Runtime runtime = Runtime.getRuntime();
-			if (osName.startsWith("Mac OS")) {
-				runtime.exec("open " + url);
-			} else if (osName.startsWith("Windows"))
-				runtime.exec("rundll32 url.dll,FileProtocolHandler " + url);
-			else {
-				// assume Unix or Linux
-				String[] browsers = { "firefox", "opera", "konqueror", "epiphany", "mozilla", "netscape" };
-				String browser = null;
-				for (int count = 0; count < browsers.length && browser == null; count++)
-					if (runtime.exec(new String[] { "which", browsers[count] }).waitFor() == 0)
-						browser = browsers[count];
+    public static void open(String url) {
+        String osName = System.getProperty("os.name");
+        Runtime runtime = Runtime.getRuntime();
+        try {
+            if (osName.startsWith("Mac OS")) {
+            	runtime.exec("open " + url);
+            } else if (osName.startsWith("Windows"))
+            	runtime.exec("rundll32 url.dll,FileProtocolHandler " + url);
+            else {
+                //assume Unix or Linux
+                String[] browsers =
+                        {"firefox", "opera", "konqueror", "epiphany", "mozilla", "netscape"};
+                String browser = null;
+                for (int count = 0; count < browsers.length && browser == null; count++)
+                    if (Runtime.getRuntime().exec(new String[]{"which", browsers[count]}).waitFor() == 0)
+                        browser = browsers[count];
 
-				if (browser == null)
-					throw new Exception(String.format("Could not find web browser.%nURL: %s", url));
-				else
-					runtime.exec("xdg-open " + url);
-			}
-		} catch (Exception e) {
-			JOptionPane.showMessageDialog(null, errMsg + ":\n" + e.getLocalizedMessage());
-		}
-	}
-}
+                if (browser == null)
```

Comment:
I know that we don't have style guide yet (something that I'll try to fix), but for the time being, please add braces to the one-liner `if` constructs. I've been trying to do that whenever I met them :)

Reply:
Done